src/gobimportclient/import_client.py

```diff
@@ -102,6 +128,29 @@ def publish(self):
             timestamp=datetime.datetime.now().isoformat()
         )
 
-        import_message = ImportMessage.create_import_message(metadata.as_header, None, self._gob_data)
+        summary = {
+            'RECORDS': len(self._gob_data)
```

Comment:
Why capitals? Isn't nr_records a more intuitive name?

Reply:
It was matched to look like the ADD, DELETE, MODIFY and CONFIRM events that are being logged in the update and compare step. For the logging purposes let's change them to:
num_records
num_added
num_confirmed
num_deleted
num_modified

@Vegter do you agree?

Reply:
Great!